dimod/quadratic/quadratic_model.py

```diff
@@ -398,20 +398,42 @@ def variables(self) -> Variables:
         return self.data.variables
 
     @forwarding_method
-    def add_linear(self, v: Variable, bias: Bias):
+    def add_linear(self, v: Variable, bias: Bias, *,
+                   default_vartype=None,
+                   default_lower_bound=None,
+                   default_upper_bound=None,
+                   ):
         """Add a linear bias to an existing variable.
```

Comment:
For completeness, could update the docstring summary:

```suggestion
        """Add a linear bias to an existing variable or a new variable with
        specified vartype.
```